release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3360,6 +3360,49 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.42
+[id="ocp-4-16-42_{context}"]
+=== RHSA-2025:8556 - {product-title} {product-version}.42 bug fix and security update
+
+Issued: 11 June 2025
+
+{product-title} release {product-version}.42 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:8556[RHSA-2025:8556] advisory. 
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.42 --pullspecs
+----
+
+[id="ocp-4-16-42-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the console alerted users that compute nodes must be updated within 60 days when performing a control plane only update. With this update, the console no longer displays this invalid alert. (link:https://issues.redhat.com/browse/OCPBUGS-56858[OCPBUGS-56858])
```

Comment:
```suggestion
* Previously, the web console alerted users that compute nodes must be updated within 60 days when performing a control plane only update. With this update, the web console no longer displays this invalid alert. (link:https://issues.redhat.com/browse/OCPBUGS-56858[OCPBUGS-56858])
```
It is actually important to distinguish this console from other things, such as the hybrid cloud console.